app.js

```diff
@@ -1,24 +1,30 @@
 const express = require('express');
 const app = express();
-const { getTopics} = require("./controll/controller")
+const { getTopics, getApi} = require("./controll/controller")
 
 
-app.use(express.json());
-
 app.get('/api/topics', getTopics)
 
+app.get('/api', getApi)
+
 //route for handling all invalid route
 app.get("*", (req, res, next) => {
     res.status(404).send({msg : "Not found"})
 })
 
-app.use((err, req, res, next) => {
-    if(err.statusCode && err.msg){
-        res.status(err.statusCode).send({msg: err.msg})
-    } else {
-        next(err);
-    }
-})
+
+// app.use((err, req, res, next) => {
+//     console.log(err)
+//     if(err.statusCode && err.msg){
+//         res.status(err.statusCode).send({msg: err.msg})
+//     } else {
+//         next(err);
+//     }
+// })
+
```

Comment:
It's ok at this point to have the 500 catch all error handling middleware function uncommented and already doing it's job